lib/ueberauth/strategy.ex

```diff
@@ -165,7 +173,7 @@ defmodule Ueberauth.Strategy do
   alias Ueberauth.Auth.Info
   alias Ueberauth.Auth.Extra
 
-  @state_param_cookie_name "ueberauth.state_param"
+  @anti_csrf_token_cookie_name "ueberauth.anti_csrf_token"
```

Comment:
What was the reason for the renaming? I am confused by the usage of the word `anti`

Reply:
I just wanted to distinguish custom state data and CSRF token name. We could change that name to a simpler one - e.g. `csrf_token", it's a good idea.

Reply:
What would be the issue with `state` param?

CSRF prevention could be done using the `state` param but the data is about the `state` param, so I am failing to understand the renaming.

Please help.

Reply:
I don't want to pass custom data (stored in providers state param) to that cookie - but only the CSRF token. And that is the reason for the rename as the cookie is only needed to store CSRF token (not whole state param value with custom data related to application).